server/app/views/applicant/ProgramIndexView.java

```diff
@@ -106,20 +108,33 @@ public Content render(
     bundle.setTitle(messages.at(MessageKey.CONTENT_GET_BENEFITS.getKeyName()));
     bannerMessage.ifPresent(bundle::addToastMessages);
     bundle.addMainContent(
-        topContent(
-            messages.at(MessageKey.CONTENT_GET_BENEFITS.getKeyName()),
-            messages.at(
-                MessageKey.CONTENT_CIVIFORM_DESCRIPTION.getKeyName(), civicEntityShortName)),
+        topContent(messages, userName),
         mainContent(
             request, messages, applicationPrograms, applicantId, messages.lang().toLocale()));
 
     return layout.renderWithNav(request, userName, messages, bundle, /*includeAdminLogin=*/ true);
   }
 
-  private DivTag topContent(String titleText, String infoTextLine) {
-    // "Get benefits"
+  private DivTag topContent(Messages messages, Optional<String> userName) {
+
```

Comment:
**nit (optional):** We don't generally have a blank line after a function declaration in the codebase